sormas-ui/src/main/java/de/symeda/sormas/ui/events/eventParticipantMerge/PickLeadEventParticipant.java

```diff
@@ -0,0 +1,158 @@
+package de.symeda.sormas.ui.events.eventParticipantMerge;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import com.vaadin.server.ExternalResource;
+import com.vaadin.ui.Component;
+import com.vaadin.ui.HorizontalLayout;
+import com.vaadin.ui.Label;
+import com.vaadin.ui.Link;
+import com.vaadin.ui.VerticalLayout;
+import com.vaadin.v7.ui.CustomField;
+
+import de.symeda.sormas.api.event.EventParticipantSelectionDto;
+import de.symeda.sormas.api.event.EventReferenceDto;
+import de.symeda.sormas.api.i18n.Captions;
+import de.symeda.sormas.api.i18n.I18nProperties;
+import de.symeda.sormas.api.i18n.Strings;
+import de.symeda.sormas.api.utils.DataHelper;
+import de.symeda.sormas.ui.SormasUI;
+import de.symeda.sormas.ui.events.EventDataView;
+import de.symeda.sormas.ui.utils.CssStyles;
+import de.symeda.sormas.ui.utils.VaadinUiUtil;
+
+public class PickLeadEventParticipant extends CustomField<List<String>> {
+
+	private Map<String, EventParticipantGridByEvent> duplicateEventParticipantByEventGridMap;
```

Comment:
this can be changed into a list named `eventParticipantSelectGrids`

Reply:
variable name changed